From 090bca0f16c1b550ef7a450eb4d1e28a7557359f Mon Sep 17 00:00:00 2001
From: Guo Xuenan <guoxuenan@huawei.com>
Date: Thu, 21 Oct 2021 21:06:00 +0800
Subject: [PATCH] make OPTIMIZE_INLINING config editable
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 090bca0f16c1b550ef7a450eb4d1e28a7557359f


hulk inclusion
category: bugfix
bugzilla: 182617 https://gitee.com/openeuler/kernel/issues/I4DDEL

--------------------------------

in commit e2b3eb3c5 disable OPTIMIZE_INLINING by default, make it
editable when using menuconfig

Signed-off-by: Guo Xuenan <guoxuenan@huawei.com>
Reviewed-by: Jason Yan <yanaijie@huawei.com>
Signed-off-by: Chen Jun <chenjun102@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 lib/Kconfig.debug | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index a1a835c5d6cc..6ce7350d6b69 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -334,6 +334,7 @@ config HEADERS_INSTALL
 	  as uapi header sanity checks.
 
 config OPTIMIZE_INLINING
+	bool "Optimize inlining"
 	def_bool n
 	help
 	  This option determines if the kernel forces gcc to inline the functions
-- 
2.26.2

